internal/controller/eventbus/controller.go

```diff
@@ -327,11 +328,14 @@ func (ctrl *controller) DeleteEventbus(ctx context.Context, eb *wrapperspb.UInt6
 	return &emptypb.Empty{}, nil
 }
 
-func (ctrl *controller) deleteEventbus(ctx context.Context, id vanus.ID) error {
+func (ctrl *controller) deleteEventbus(ctx context.Context, id vanus.ID, system bool) error {
 	bus, exist := ctrl.eventbusMap[id]
 	if !exist {
 		return errors.ErrResourceNotFound.WithMessage("the eventbus doesn't exist")
 	}
+	if !system && strings.HasPrefix(bus.Name, primitive.SystemEventbusNamePrefix) {
+		return errors.ErrResourceCanNotOp.WithMessage("system eventbus can't delete")
```

Comment:
There is a scenario of deleting system eventbus in timer

Reply:
Or separately provide a DeleteSystemEventbus interface for internal modules to use ？

Reply:
I will remove the limit and add an issue to trace the problem

Reply:
I have removed the limit and added a comment todo, and also I added an issue #575 to trace it